GRFC-TEMPLATE.md

```diff
@@ -2,21 +2,19 @@ Title
 ----
 * Author(s): [Author Name, Co-Author Name ...]
 * Approver: a11r
-* Status: {Draft, In Review, Ready for Implementation, Implemented}
```

Comment:
I'm thinking that we should probably go through the existing gRFCs and remove the "Status:" field, since it's pretty useless and misleading.  But we can do that in a separate PR.

Reply:
Agreed.  I was thinking of the same, but maybe we should wait a month or two, since there are PRs pending with Status fields in them as well.